<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceForm</class>
 <widget class="QWidget" name="InterfaceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="grpVisualizer">
     <property name="title">
      <string>Visualizer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="4">
         <widget class="QCheckBox" name="chkZBuffer">
          <property name="text">
           <string>Z-buffer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="radMSAA">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>MSAA</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Line width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="txtLineWidth">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>9.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>FPS lock:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboFps">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>120</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Gcode Draw Mode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QRadioButton" name="radDrawModeVectors">
          <property name="text">
           <string>Vectors</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QRadioButton" name="radDrawModeRaster">
          <property name="text">
           <string>Raster</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4">
         <widget class="QRadioButton" name="radGrayscaleZ">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>By 'Z'-code</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QRadioButton" name="radGrayscaleS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>By 'S'-code</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="chkGrayscale">
          <property name="text">
           <string>Grayscale Segments</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QDoubleSpinBox" name="txtSimplifyPrecision">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="chkSimplify">
          <property name="text">
           <string>Simplify Geometry</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="lblSimpilyPrecision">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Precision</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="chkVSync">
          <property name="text">
           <string>V-sync</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="chkAntialiasing">
          <property name="text">
           <string>Anti Aliasing</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpConsole">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Console</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="chkShowProgramCommands">
          <property name="text">
           <string>Show G-code Program Commands</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowUICommands">
          <property name="text">
           <string>Show UI Commands</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkAutocompletion">
          <property name="text">
           <string>Command Auto-Completion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0">
        <item row="7" column="2">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Z-Movement</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Toolpath</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QWidget" name="clpToolpathZMovement" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QWidget" name="clpToolpathHighlight" native="true"/>
        </item>
        <item row="8" column="1">
         <widget class="QWidget" name="clpToolpathStart" native="true"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Highlight:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Start Point</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QWidget" name="clpToolpathEnd" native="true"/>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>End Point</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="clpVisualizerBackground" native="true"/>
        </item>
        <item row="2" column="3">
         <widget class="QWidget" name="clpVisualizerText" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Visualizer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Tool</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="clpTool" native="true"/>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Drawn</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QWidget" name="clpToolpathDrawn" native="true"/>
        </item>
        <item row="6" column="1">
         <widget class="QWidget" name="clpToolpathNormal" native="true"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Normal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
